setup.py

```diff
@@ -17,6 +17,14 @@
 with open("snowfakery/version.txt") as f:
     version = f.read().strip()
 
+packages = [
+    p
+    for p in setuptools.find_namespace_packages()
+    if p.startswith("snowfakery") and not p.startswith("snowfakery.docs")
+]
+
+print("Releasing packages", packages)
```

Comment:
```suggestion
```

Seems like an anti-pattern to use print here, and we certainly can't assume that someone is releasing something just because they're running setup.py